village/src/components/SmurfForm.js

```diff
@@ -10,21 +10,25 @@ class SmurfForm extends Component {
     };
   }
 
-  addSmurf = event => {
-    event.preventDefault();
-    // add code to create the smurf using the api
-
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
-    });
-  }
 
   handleInputChange = e => {
     this.setState({ [e.target.name]: e.target.value });
   };
+  addSmurf = () => {
+    const name = {
+      name: this.state.name,
+      age: this.state.age,
+      height: this.state.height
```

Comment:
You could make this a little shorter and perhaps even easier to read with [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):
```javascript
addSmurf = () => {
  const { name, age, height } = this.state;
  axios
    .post('http://localhost:3333/smurfs', { name, age, height })
    .then( // ...
};
```

Reply:
or, alternatively, since you know what's on your state, you _could_ do this:
```javascript
addSmurf = () => {
  axios
    .post('http://localhost:3333/smurfs', this.state)
    .then( // ...
};
```